### ISSUE TITLE
Rpi 1  - Error on build message

### ISSUE THREAD

**Issue by Newan** [rytilahti/python-miio#197]
I installed always like the doc. Dicovery works and find my vaccum robo. Set Env for IP and Token. But status an start not work:

```
pi@raspberrypi:~ $ mirobo
Traceback (most recent call last):
  File "/usr/local/bin/mirobo", line 11, in <module>
    sys.exit(cli())
  File "/home/pi/.local/lib/python3.5/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/pi/.local/lib/python3.5/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/pi/.local/lib/python3.5/site-packages/click/core.py", line 1043, in invoke
    return Command.invoke(self, ctx)
  File "/home/pi/.local/lib/python3.5/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/pi/.local/lib/python3.5/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/pi/.local/lib/python3.5/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/home/pi/.local/lib/python3.5/site-packages/miio/vacuum_cli.py", line 89, in cli
    ctx.invoke(status)
  File "/home/pi/.local/lib/python3.5/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/pi/.local/lib/python3.5/site-packages/click/decorators.py", line 64, in new_func
    return ctx.invoke(f, obj, *args[1:], **kwargs)
  File "/home/pi/.local/lib/python3.5/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/pi/.local/lib/python3.5/site-packages/miio/vacuum_cli.py", line 125, in status
    res = vac.status()
  File "/home/pi/.local/lib/python3.5/site-packages/miio/vacuum.py", line 108, in status
    return VacuumStatus(self.send("get_status")[0])
  File "/home/pi/.local/lib/python3.5/site-packages/miio/device.py", line 216, in send
    m = Message.build(msg, ctx)
TypeError: build() takes 2 positional arguments but 3 were given
```

**Comment by yawor** [comment]
It's a known issue caused by a change in construct library. You can downgrade the construct or wait for new release of python-miio. You can also download latest sources from the repo as the fix has already been merged.

**Comment by rytilahti** [comment]
This is indeed fixed by #192.
